theme_monglia/static/src/snippets/s_numbers/000.scss

```diff
@@ -1,11 +1,3 @@
-.s_numbers {
-    .s_number {
-        font-size: 75px;
-        @include media-breakpoint-down(md) {
-            font-size: 60px;
-        }
-    }
-    h6 {
-        text-transform: uppercase;
-    }
+.s_numbers h6 {
+    text-transform: uppercase;
```

Comment:
Keep the `.s_numbers` selector alone. That way you'll preserve history but it's also a better convention for our snippet CSS files. (In general, it's good to regroup but I would keep the name of the snippet alone at the top).